templates/default/src/main/liberty/config/configDropins/defaults/quick-start-security.xml

```diff
@@ -0,0 +1,3 @@
+<server>
+    <quickStartSecurity userName="admin" userPassword="adminpwd" />
```

Comment:
Should we use the generated keystore password?

Reply:
Updated this issue to track this change: https://github.com/OpenLiberty/application-stack/issues/21